### PR TITLE
adding alb

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,6 +327,60 @@ this repository will be candidate for a rewrite.
 
 ### Example
   ```
+  module "alb" {
```
Comment (SamClinckspoor) [inline README.md]: can you put the example under `## alb_with_ssl_no_s3logs`?

**Comment by MattiasGees** [comment]
I would only add one health check and one target group. Sometimes you only need ssl as the redirect from http to https can be handled from cloudfront if there is a cloud front in front of it. I think we should make it possible to point one target group at multiple listeners (e.g. the HTTP and HTTPS listener are sending their requests to the same target group).

**Comment by venturel** [comment]
you can create a separate module if you want for that or I just use this for FLAVR and I don't make it generic. I honestly don't like your approach because it make it hard to understand if the traffic is http or https. I don't think it's actually in the scope of this PR

**Comment by MattiasGees** [comment]
If we merge this we will block ourselves for a lot of other projects.

**Comment by ringods** [comment]
@MattiasGees can you elaborate a bit how we will block ourselves then? Isn't @venturel his approach a repetition of the ELB approach already there?

**Review by SamClinckspoor — APPROVED** [review]

**Review by simonrondelez — APPROVED** [review]

**Pull request merged**